internal/command/upgrade_db.go

```diff
@@ -29,9 +37,10 @@ type upgradeDbCmd struct {
 func newUpgradeDatabaseCmd() *upgradeDbCmd {
 	cmd := upgradeDbCmd{
 		Command: cobra.Command{
-			Use:               "db",
+			Use:               "db [POSTGRES_URL]",
```

Comment:
shouldn't this be using `envVarPSQLURL` too?

Reply:
It supports both, the env var and passing the URL as parameter, same "baur init db"